app/models/ems_refresh/save_inventory.rb

```diff
@@ -101,7 +102,11 @@ def save_vms_inventory(ems, hashes, target = nil)
         found.raw_power_state = key_backup[:raw_power_state]
 
         link_habtm(found, key_backup[:storages], :storages, Storage)
-        link_habtm(found, key_backup[:security_groups], :security_groups, SecurityGroup)
+        # TODO(lsmola) NetworkManager, once all providers are converted, security groups relation of Vm will go away,
+        # being moved to NetworkPort, we will ignore here through associations
+        if found.try(:security_groups).try(:proxy_association).try(:reflection).try(:options).try(:[], :through) != :network_ports
```

Comment:
@matthewd hm if this is the bad way of inspecting associations, maybe we should refactor also this code https://github.com/Ladas/manageiq/blob/75bb48d5edcc708db7d3e1023707870560e6f471/app/helpers/textual_summary_helper.rb#L116-116 ?